ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1056,11 +1056,11 @@ tests:
         MIRROR_IMAGES=true
         INSTALLER_PROXY=true
         NETWORK_TYPE=OVNKubernetes
-      E2E_RUN_TAGS: '@amd64 and @baremetal-ipi'
+      E2E_RUN_TAGS: '@amd64 and @baremetal-ipi and @disconnected'
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
-- as: baremetalds-ipi-ovn-dualstack-primaryv6-f28-destructive
+- as: baremetalds-ipi-ovn-dualstack-primaryv6-disco-f28-destructive
```

Comment:
```suggestion
- as: baremetalds-ipi-ovn-dualstack-primaryv6-disc-f28-destructive
```

Reply:
Also suggest to update `baremetalds-ipi-ovn-dualstack-primaryv6-disconnected-fips-f7` to `baremetalds-ipi-ovn-dualstack-primaryv6-disc-fips-f7`, because they are a pair.

Reply:
updated. thanks.